src/porepy/utils/permutations.py

```diff
@@ -29,6 +31,8 @@ def multinary_permutations(base, length):
         array, size length: Array describing the next number combination.
 
     """
+    msg = "This functionality is deprecated and will be removed in a future version"
+    warnings.warn(msg, DeprecationWarning)
```

Comment:
I have been wanting to do this for years, but replacing the single usage of this function is not straightforward. There should be built-in functions that can do this though, so let's deprecate it.

Reply:
So, to be clear, are you suggesting to keep the warning and deal with how to find a substitute at a later point?

Reply:
Yes.